inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/load/KafkaLoadNode.java

```diff
@@ -226,8 +226,8 @@ public String getMetadataKey(MetaField metaField) {
             case OP_TYPE:
                 metadataKey = "value.op-type";
                 break;
-            case DATA:
-                metadataKey = "value.data";
+            case DATA_CANAL:
```

Comment:
Maybe it is better to add 'source.multiple.format' to decide which serialization to use？

Reply:
I guess using two seperated meta data is fare enough?